landing-pages/site/content/en/ecosystem/_index.md

```diff
@@ -57,6 +57,8 @@ If you would you like to be included on this page, please reach out to the [Apac
 
 [Airflow ECR Plugin](https://github.com/asandeep/airflow-ecr-plugin) - Plugin to refresh AWS ECR login token at regular intervals. This is helpful where DockerOperator needs to pull images hosted on ECR.
 
+[Airflow OpenMLDB Provider](https://github.com/4paradigm/OpenMLDB/tree/main/extensions/airflow-provider-openmldb) - Airflow OpenMLDB Provider containing Operators for Feature Extraction on OpenMLDB.
```

Comment:
I wonder, If we going to add some others provider or plugin to ecosystem(it maybe a huge number of it), should we change to other format instead of list? For more easy to maintainer or search by our users @potiuk

Reply:
Feel free to propose it @zhongjiajie